.travis.yml

```
@@ -20,17 +20,17 @@ matrix:
       env: TOXENV=py37
     - python: 3.8
       env: TOXENV=py38
-    - python: 3.8-dev
-      env: TOXENV=py38
+    - python: 3.9-dev
```

Python 3.9 final is now available:
```suggestion
    - python: 3.9
```